atlp-rwanda/e-comm-team-brogrammers-fn
# #184948162 seller get one or all products from their collection

#### What does this PR do?

this PR is for  seller to be able to view their own collection as well as getting one product from their collection

#### Description of Task to be completed?

-this task includes to create UI for both view all products in collection as well as  get one product from collection
-fetch APIs for those endpoints

#### How should this be manually tested?

-first login and got to _avatar_ of the user and click on  _your collection_
-as a seller you will see your profile information and all products in your collection
- click on any product if you want to get more details about it

#### Any background context you want to provide?

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184948162

#### Screenshots (if appropriate)

<img width="1008" alt="Screenshot 2023-05-02 at 18 40 13" src="https://user-images.githubusercontent.com/99663528/235730370-b593c9a2-265c-4be3-93f8-da82c84221af.png">
<img width="994" alt="Screenshot 2023-05-02 at 18 41 40" src="https://user-images.githubusercontent.com/99663528/235730418-548e1b37-97dc-415b-a24e-1b39a497698f.png">


#### Questions:
